vectorstores/marqo.py

```diff
@@ -22,15 +22,13 @@ def __init__(self):
         self.collection_name = os.environ["VECTOR_COLLECTION_NAME"]
         self.embedding_model = os.environ["EMBEDDING_MODEL"]
         self.index_settings = {
-            "index_defaults": {
-                "treat_urls_and_pointers_as_images": False,
-                "model": self.embedding_model,
-                "normalize_embeddings": True,
-                "text_preprocessing": {
-                    "split_length": self.SPLIT_LENGTH,
-                    "split_overlap": self.SPLIT_OVERLAP,
-                    "split_method": "sentence"
-                }
+            "treatUrlsAndPointersAsImages": False,
```

Comment:
`index_defaults`  not needed?

Reply:
yes. this is the main change (or we can say upgrade)